build.sh

```diff
@@ -1,23 +1,23 @@
 #!/bin/bash -eux
 
-set -a
-
-name=conjur
-
-CONJUR_VERSION=0.1.0-stable
-CONJUR_IMAGE=quay.io/conjur/${name}:${CONJUR_VERSION}
-CONJUR_TARBALL=${name}_${CONJUR_VERSION}.tar.gz
-
+CONJUR_VERSION="${1:-latest}"
 # Default these variables to make development easier. They should be get in the environment (i.e. in the Jenkinsfile)
 # when this script gets used as part of a build.
 : ${INSTANCE_ID_FILE=dev-EC2.txt}
 : ${AMI_ID_FILE=dev-AMI.txt}
 
 finish() {
   if [ -f "${INSTANCE_ID_FILE}" ]; then
-    ./ansible.sh ${@-ansible-playbook -v -e instance_id="$(< ${INSTANCE_ID_FILE})" -e instance_state=absent instance.yml}
+    ./ansible.sh ansible-playbook -v \
+      -e instance_id="$(< ${INSTANCE_ID_FILE})" \
+      -e instance_state=absent \
+      instance.yml
   fi
 }
 trap finish EXIT
 
-./ansible.sh ${@-ansible-playbook -vvv -e ami_id_filename="${AMI_ID_FILE}" -e instance_id_filename="${INSTANCE_ID_FILE}" build.yml}
+./ansible.sh ansible-playbook -vvv \
```

Comment:
Now that this is working, we might want to dial this back to `-v`.

Reply:
good call